lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -2551,109 +2551,71 @@ def model_add_psz_ac(model,
         clg_coil = nil
       end
 
-      # wrap coils in a unitary system if cycling
+      # cycling fan
       if fan_type == 'Cycling'
```

Comment:
this if / else statement is a mess.  It include fan_type and fan_location.  these should be independent

Reply:
Last time, I kept the if/else statement like the previous version, but we agree that it is better to avoid a mess; so, I address your comment and commit the branch. 
There are still two questions:
(1) in Lines 2613 and 2617, there are still messes. Should we move these two lines out or keep the current way?
(2) Like Jeremy mentioned before, we noticed that there's a 90.1-2013 PRM test failure, which seems expected since we're now using unitary systems for PSZ systems. You mentioned updating/adding some of the tests, we're not sure if you were planning on updating this one.
Thanks.